phpmetrics.json

```diff
@@ -0,0 +1,164 @@
+{
+    "includes": [
+        "src"
+    ],
+    "exclude": [
+      "tests"
+    ],
+    "report": {
+        "html": "htmlout",
+        "csv": "/tmp/report.csv",
+        "json": "/tmp/report.json",
+        "violations": "/tmp/violations.xml"
+    },
+    "groups": [
```

Comment:
Please use readable names for the "name" key.

Either the namespace suffix, ie: SessionPsr16 or the directory name, ie: session-psr16

Reply:
All names have been changed to reflect the namespace suffix.